packages/hint-validate-summary/README.md

```diff
@@ -0,0 +1,88 @@
+# validate-summary (`@hint/hint-validate-summary`)
+
+Summary tag should have display: list-item only.
+
+## Why is this important?
+
+Changing the display on a summary tag to anything other than list-item
+will cause the open/closeicon to disappear in browsers which implement
+the latest version of the standard.(eg: Firefox).
+Chrome has
+<a href="https://bugs.chromium.org/p/chromium/issues/detail?id=590014&q=component%3ABlink%3EHTML%3EDetails%20&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified">
```

Comment:
Let's use a markdown link reference here:
```suggestion
[a bug open][chromium-summary-issue]
```

Then down in "Link Labels" near the end of the file you can add:
```md
[chromium-summary-issue]: https://bugs.chromium.org/p/chromium/issues/detail?id=590014
```